### PR TITLE
SNP-sites tool

### PR DESCRIPTION
**Pull request by Eugene79**
Source: https://github.com/sanger-pathogens/snp-sites
Conda recipe provided by authors

### CODE REVIEW

**Comment by bgruening** [comment]
@Eugene79 please do not add commits to this branch for other tools. If you want to update a tool, use a different branch. For this PR it is ok, no need to revert this, but please remember this the next time. Thanks.

**Comment by bgruening** [comment]
@Eugene79 for the help section I found this site very nice to use: http://rst.ninjs.org
I tried to make it a little bit more beautiful and removed the citation as this is already added with `<citation>`.

File: tools/snp-sites/snp-sites.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id='snp_sites' name='Finds SNP sites from a multi-FASTA alignment file' version='0.1.0'>
```
Comment (abretaud) [inline tools/snp-sites/snp-sites.xml]: I would change this to `name="Find SNP sites"` then `<description>from a multi-FASTA alignment file</description>`

File: tools/spades/spades.xml [hunk 2]
```diff
@@ -212,7 +212,7 @@ for i,line in enumerate(sys.stdin):
 
 SPAdes – St. Petersburg genome assembler – is intended for both standard isolates and single-cell MDA bacteria assemblies. See http://bioinf.spbau.ru/en/spades for more details on SPAdes.
 
-This wrapper runs SPAdes 3.9, collects the output, and throws away all the temporary files. It also produces a tab file with contig names, length and coverage.
+This wrapper runs SPAdes 3.10, collects the output, and throws away all the temporary files. It also produces a tab file with contig names, length and coverage.
```
Comment (nsoranzo) [inline tools/spades/spades.xml]: I'd just remove the version, no reason to have it here.

File: tools/snp-sites/snp-sites.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+.. code-block::
+
+  >sample1
+  AGACACAGTCAC
+
+  >sample1
+  AGACAC----AC
+
+  >sample1
+  AAACGCATTCAN
```
Comment (Eugene79) [inline tools/snp-sites/snp-sites.xml]: Included and explained briefly a VCF output.

File: tools/spades/spades.xml (hunk 2 above)
Comment (Eugene79) [inline tools/spades/spades.xml]: Done

File: tools/snp-sites/snp-sites.xml (hunk 1 above)
Comment (Eugene79) [inline tools/snp-sites/snp-sites.xml]: Done

**Comment by bgruening** [comment]
Super cool! Thanks @Eugene79!

File: tools/snp-sites/snp-sites.xml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+-----   --- --  --- ---    ----    ------  ----    ------  ------- ------- -------
+1       8   .   G   .,T    .       .       .       GT      0       1       2
+=====   === ==  === ===    ====    ======  ====    ======  ======= ======= =======
+
+Thus the tool identified three variations (SNPs): in 2nd, 5th, and 8th positions (A instead of G, G instead of A, and unknown nucleotide or T instead of G, respectively).
+
+
+	]]>
+    </help>
+	<citations>
```
Comment (nsoranzo) [inline tools/snp-sites/snp-sites.xml]: Indentation here is wrong (uses a tab instead of 4 spaces).

**Pull request merged**